### ISSUE TITLE
installing requirements for viff and brainglance fails

### ISSUE THREAD

**Issue by amwink** [lipsia-fmri/vini#1]
After downloading the viff and brainglance repositories, I found in the installation guides that the only command necessary is

  **`pip install -r requirements.txt`**

but when I do that I get this error for both packages:

**`Collecting sklearn==0.20.2 (from -r requirements.txt (line 3))`**
**`  Could not find a version that satisfies the requirement sklearn==0.20.2 (from -r requirements.txt (line 3)) (from versions: 0.0)`**
**`No matching distribution found for sklearn==0.20.2 (from -r requirements.txt (line 3))`**

However, when I load sklearn in ipython it's just there:

**`In [1]: import sklearn`**

**`In [2]: sklearn.show_versions()`**

**`System:`**
**`    python: 2.7.15+ (default, Feb  3 2019, 13:13:16)  [GCC 8.2.0]`**
**`   machine: Linux-4.9.0-8-amd64-x86_64-with-debian-9.8`**
**`executable: /usr/bin/python`**
**`BLAS:`**
  **`    macros: HAVE_CBLAS=None`**
  **`cblas_libs: openblas, openblas`**
  **`  lib_dirs: /usr/lib/x86_64-linux-gnu`**
**`Python deps:`**
  **`    Cython: 0.29.2`**
  **`     scipy: 1.1.0`**
  **`setuptools: 33.1.1`**
  **`       pip: 10.0.1`**
  **`     numpy: 1.16.1`**
  **`    pandas: 0.23.3`**
  _**`   sklearn: 0.20.2`**_

In other words, the version required by viff / brainglance is there. Do you have any idea where it clashes?

**Comment by lipsia-fmri** [comment]
thanks for pointing this out!
the issue is that we used "sklearn" instead of "scikit-learn" in our requirements. While the package is called sklearn, you need to install it via "pip install scikit-learn". We fixed the issue in the requirements.
